app/scripts/components/exploration/index.tsx

```diff
@@ -17,12 +17,20 @@ import PageHero from '$components/common/page-hero';
 import { DATASETS_PATH, EXPLORATION_PATH } from '$utils/routes';
 
 /**
+ * Container component that manages exploration view routing and data state.
+ *
+ * Routes between two view modes based on URL parameter:
+ * - Simple view (`?viewMode=simple`): Minimal interface for embedding
+ * - Default view: Full exploration and analysis interface
+ *
  * @LEGACY-SUPPORT
  *
- * @NOTE: This container component serves as a wrapper for the purpose of data management, this is ONLY to support current instances.
- * veda2 instances can just use the direct component, 'ExplorationAndAnalysis', and manage data directly in their page views
+ * @NOTE: This container component serves as a wrapper for the purpose of data management,
+ * this is ONLY to support current instances. veda2 instances can just use the direct
```

Comment:
@vgeorge what is current instances vs veda2? is current instances, veda as a submodule? and veda2 is the the next-veda-ui implementation as a library (instead of submodule)?

when you say current instances, if I read this in one year, will that still apply?